DuckDuckGo/Preferences/Model/SyncPreferences.swift

```diff
@@ -21,7 +21,7 @@ import DDGSync
 import Combine
 import Common
 import SystemConfiguration
-import SyncUI
+import SyncUI_macOS
```

Comment:
Swift resolves most characters to an underscore when determining module name, so imports are different from the package & target name.